images/driver/Dockerfile

```diff
@@ -48,6 +48,21 @@ RUN go build -a -ldflags="-w -s -extldflags=static -X sigs.k8s.io/vsphere-csi-dr
 FROM ${BASE_IMAGE}
 LABEL "maintainers"="Divyen Patel <divyenp@vmware.com>, Sandeep Pissay Srinivasa Rao <ssrinivas@vmware.com>, Xing Yang <yangxi@vmware.com>"
 
+# install nfs-utils
+# Description   : The nfs-utils package contains simple nfs client service
+RUN tdnf -y install nfs-utils
+
+# install util-linux
+# Description   : Utilities for handling file systems, consoles, partitions,
+RUN tdnf -y install util-linux
```

Comment:
Each `RUN` command is creating a layer. Can we just club them into a single `RUN` command?

Reply:
We can club RUN command for installing packages. I will do that.